.travis.yml

```diff
@@ -87,10 +87,11 @@ matrix:
 
 # Dependencies installation commands
 install:
+  - pip install -U pip
   # codecov is the interface to codecov.io; see after_success
-  - "echo 'travis_fold:start:codecov'; pip install codecov; echo 'travis_fold:end:codecov'"
-  - "echo 'travis_fold:start:tw'; [ $TWISTED = latest ] || pip install Twisted==$TWISTED; echo 'travis_fold:end:tw'"
-  - "echo 'travis_fold:start:sa'; [ $SQLALCHEMY = latest ] || pip install sqlalchemy==$SQLALCHEMY; echo 'travis_fold:end:sa'"
+  - pip install codecov
```

Comment:
Travis automatically folds individual install steps, so folding again inside the step is a waste (and just makes seeing the contents of the fold more annoying).